cdap-app-templates/cdap-etl/cdap-etl-lib/src/test/java/co/cask/cdap/template/etl/common/StructuredRecordStringConversionTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.template.etl.common;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.api.data.schema.Schema;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test for {@link StructuredRecordStringConversion} class from {@link StructuredRecord} to json string
+ */
+@SuppressWarnings("unchecked")
+public class StructuredRecordStringConversionTest {
+  Schema schema;
+
+  @Test
+  public void checkConversion() {
+    StructuredRecord initial = getStructuredRecord();
+    String jsonOfRecord = StructuredRecordStringConversion.toJsonString(initial);
+    StructuredRecord recordOfJson = StructuredRecordStringConversion.fromJsonString(jsonOfRecord, schema);
+    areRecordsEqual(initial, recordOfJson);
```

Comment:
the 'are' seems like it returns a boolean.
Perhaps rename to `assertRecordsEqual`